lib/rspec/collection_matchers/have.rb

```diff
@@ -116,8 +116,8 @@ def description
         "have #{relative_expectation} #{@collection_name}"
       end
 
-      def respond_to?(m)
-        @expected.respond_to?(m) || super
+      def respond_to?(m, include_all: false)
```

Comment:
Ah, I've just realised you've used a keyword argument here, that's not going to work on all versions of Ruby that this supports... maybe try using a splat and just passing through the arguments, or is the official definition with keyword args? If so you'll have to define multiple versions I suspect.